music21/sorting.py

```diff
@@ -153,21 +156,16 @@ def shortRepr(self):
         >>> st.shortRepr()
         'End <4.7.[Grace].200>'
         '''
-        reprParts = []
```

Comment:
revert since not performance critical and later method is much less readable.

Reply:
I disagree about the readability, but no problem to revert this chunk.